Hackney.Shared.HousingSearch/Hackney.Shared.HousingSearch.csproj

```diff
@@ -2,7 +2,7 @@
  <PropertyGroup>
    <TargetFramework>netstandard2.1</TargetFramework>
    <RepositoryUrl>https://github.com/LBHackney-IT/housing-search-shared</RepositoryUrl>
-   <Version>0.30.0</Version>
+   <Version>0.32.0</Version>
```

Comment:
Version number is not required as it is generated automatically, please remove the attribute entirely

Reply:
I didn't actually add the attribute initially but I'll remove it